scripts/tools/templater.js

```diff
@@ -21,11 +23,50 @@ async function validateTemplate(templateDir) {
     }
 }
 
+//pair of functions to copy a file or folder if the source exists
+function copyFileSync( source, target ) {
```

Comment:
I'd move these here https://github.com/Vermintide-Mod-Framework/Vermintide-Mod-Builder/blob/master/scripts/lib/pfs.js

Although I made it all using async functions for some reason, not sure what the thought process was back then.

Reply:
That's a good idea, I should've looked at the code base more.